packages/cli/bin/designsystemet.ts

```diff
@@ -2,13 +2,14 @@
 import { Argument, createCommand, program } from '@commander-js/extra-typings';
 import chalk from 'chalk';
 
+import type { CssColor } from '@adobe/leonardo-contrast-colors';
 import { convertToHex } from '../src/colors/index.js';
 import migrations from '../src/migrations/index.js';
 import { buildTokens } from '../src/tokens/build.js';
 import { createTokens } from '../src/tokens/create.js';
 import { writeTokens } from '../src/tokens/write.js';
 
-program.name('Designsystemet').description('CLI for working with Designsystemet').showHelpAfterError();
+program.name('designsystemet').description('CLI for working with Designsystemet').showHelpAfterError();
```

Comment:
This just fixes the `--help` output so the examples says e.g. `designsystemet` instead of `Designsystemet`, e.g.

`npx designsystemet --help`
```
Usage: designsystemet [options] [command]
```
`npx designsystemet tokens --help`
```
Usage: designsystemet tokens [options] [command]
```